store/sqlstore/post_store.go

```diff
@@ -1353,7 +1353,7 @@ func (s *SqlPostStore) determineMaxPostSize() int {
 		maxPostSize = model.POST_MESSAGE_MAX_RUNES_V1
 	}
 
-	mlog.Info(fmt.Sprintf("Post.Message supports at most %d characters (%d bytes)", maxPostSize, maxPostSizeBytes))
+	mlog.Info("Post.Message supports at most", mlog.Int("characters", maxPostSize), mlog.Int32("(bytes)", maxPostSizeBytes))
```

Comment:
The message is not clear now, I think, for this case we have to options, one is to keep the previos Sprintf to build a meaninful string, like the one was before, and adding the variables to the log anyway, something like:

```go
mlog.Info(fmt.Sprintf("Post.Message supports at most %d characters (%d bytes)", maxPostSize, maxPostSizeBytes), mlog.Int("max_characters", maxPostSize), mlog.Int32("max_bytes", maxPostSizeBytes))
```

or replace the log string for something more generic and add the variables, for example:
```go
mlog.Info("Post.Message has size restrictions", mlog.Int("max_characters", maxPostSize), mlog.Int32("max_bytes", maxPostSizeBytes))
```

Any preference/opinion here @mgdelacroix @crspeller?

Reply:
please let me know what is decided

Reply:
We generally go for the second option. Having the variables be structured is usually better.

Reply:
already done the second options